log-insights/setup/amazon-rds.md

```diff
@@ -25,16 +26,15 @@ api_key: your_api_key
 db_name: mydb
 db_username: myusername
 db_password: mypassword
-db_host: 127.0.0.1
+db_host: mydb.abc123.us-west-1.rds.amazonaws.com
 db_port: 5432
 aws_db_instance_id: your_rds_instance
 aws_region: us-west-2
 aws_access_key_id: mykey
 aws_secret_access_key: mysecret
-enable_logs: 1
```

Comment:
This was a leftover - we switched to always enabling logs when we detect a supported configuration, instead of requiring the explicit `enable_logs` setting.